File: 3726_Remove_Zeros_in_Decimal_Representation/solution.py
class Solution:
    def removeZeros(self, n: int) -> int:
        digits = []
        while n != 0:
            if n % 10 != 0:
                digits.append(n % 10)
            n //= 10

        result = 0
        for digit in reversed(digits):
            result = result * 10 + digit
        return result

File: 3726_Remove_Zeros_in_Decimal_Representation/solution.c
long long reverseNum(long long n) {
    long long num = 0;
    while (n != 0) {
        long long digit = n % 10;
        num = num * 10 + digit;
        n /= 10;
    }
    return num;
}

long long removeZeros(long long n) {
    long long num = 0;
    while (n != 0) {
        if (n % 10 != 0) {
            long long digit = n % 10;
            num = num * 10 + digit;
        }
        n /= 10;
    }
    return reverseNum(num);
}